src/main/kotlin/leets/leenk/domain/notification/domain/entity/enums/NotificationType.kt

```
@@ -0,0 +1,115 @@
+                formatted
+                    .replace("{name}", param.toString())
+                    .replace("{count}", param.toString())
+        }
+
+        return try {
+            if (formatted.contains("%")) {
+                String.format(formatted, *params)
+            } else {
+                formatted
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)